pkg/install/install.go

```diff
@@ -106,6 +117,25 @@ func RunInstall(cmd *cobra.Command, args []string) {
 	}
 }
 
+func RunUpgrade(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	system.RunSystemVersionsStatus(cmd, args)
```

Comment:
This might be confusing, printing the previous version.
We might want to add a printing, something like "current versions:"
and after the upgrade run `system.RunSystemVersionsStatus(cmd, args)` again